contracts/RestrictedNORI.sol

```diff
@@ -421,10 +408,8 @@ contract RestrictedNORI is
       revert NonexistentSchedule({scheduleId: scheduleId});
     }
     return
-      _releasedBalanceOfSingleSchedule(
-        scheduleId,
-        schedule.releasedAmountFloor
-      ) - schedule.totalClaimedAmount;
+      schedule._releasedBalanceOfSingleSchedule(totalSupply(scheduleId)) -
+      schedule.totalClaimedAmount;
```

Comment:
should probably just add another library function for this

Reply:
this is just part of the struct though